lib/lattice_field.cpp

```diff
@@ -19,8 +19,8 @@ namespace quda {
   MsgHandle* LatticeField::mh_recv_p2p_fwd[2][QUDA_MAX_DIM] { };
   MsgHandle* LatticeField::mh_recv_p2p_back[2][QUDA_MAX_DIM] { };
 
-  cudaEvent_t LatticeField::ipcCopyEvent[2][2][QUDA_MAX_DIM];
-  cudaEvent_t LatticeField::ipcRemoteCopyEvent[2][2][QUDA_MAX_DIM];
+  qudaEvent_t LatticeField::ipcCopyEvent[2][2][QUDA_MAX_DIM];
```

Comment:
IPC is very CUDA specific, and likely should be better abstracted so it's not exposed at the physics level like this.  Ok for now though.